src/applications/mhv-medical-records/actions/conditions.js

```diff
@@ -26,15 +39,28 @@ export const getConditionsList = (isCurrent = false) => async dispatch => {
 export const getConditionDetails = (
   conditionId,
   conditionList,
+  isAccelerating = false,
 ) => async dispatch => {
   try {
+    const getDetailsFunc = isAccelerating
+      ? async () => {
```

Comment:
How will this behave if you refresh the browser on the Details page? Will it fail due to `notFound`?